test/gtest/conv_embed_db.cpp

```diff
@@ -0,0 +1,215 @@
+       miopen::StartsWith(handle.GetDeviceName(), "gfx90a") || // Explicitly disabled
+       miopen::StartsWith(handle.GetDeviceName(), "gfx94") ||  // Implicitly disabled by default
+       miopen::StartsWith(handle.GetDeviceName(), "gfx103") || // Implicitly disabled by default
+       miopen::StartsWith(handle.GetDeviceName(), "gfx110") || // Implicitly disabled by default
+       GetFloatArg() != "--float")
+    {
+        GTEST_SKIP();
+    }
+    else
+    {
+        Run2dDriver(miopenFloat);
+    }
+
+#else
+    GTEST_SKIP();
+#endif
+};
+
+TEST_P(ConfigWithHalf, HalfTest)
+{
+#if MIOPEN_EMBED_DB
+
+    const auto& handle = get_handle();
+    if(miopen::StartsWith(handle.GetDeviceName(), "gfx908") || // Explicitly disabled
+       miopen::StartsWith(handle.GetDeviceName(), "gfx90a") || // Explicitly disabled
+       miopen::StartsWith(handle.GetDeviceName(), "gfx94") ||  // Implicitly disabled by default
+       miopen::StartsWith(handle.GetDeviceName(), "gfx103") || // Implicitly disabled by default
+       miopen::StartsWith(handle.GetDeviceName(), "gfx110") || // Implicitly disabled by default
+       GetFloatArg() != "--half")
+    {
+        GTEST_SKIP();
+    }
+    else
+    {
+        Run2dDriver(miopenHalf);
+    }
+
+#else
+    GTEST_SKIP();
+#endif
+};
+
+TEST_P(ConfigWithInt8, Int8Test)
+{
+#if MIOPEN_EMBED_DB
+
+    const auto& handle = get_handle();
+    if(miopen::StartsWith(handle.GetDeviceName(), "gfx908") || // Explicitly disabled
+       miopen::StartsWith(handle.GetDeviceName(), "gfx90a") || // Explicitly disabled
+       miopen::StartsWith(handle.GetDeviceName(), "gfx94") ||  // Implicitly disabled by default
+       miopen::StartsWith(handle.GetDeviceName(), "gfx103") || // Implicitly disabled by default
+       miopen::StartsWith(handle.GetDeviceName(), "gfx110") || // Implicitly disabled by default
+       GetFloatArg() != "--int8")
+    {
+        GTEST_SKIP();
+    }
+    else
+    {
+        Run2dDriver(miopenInt8);
+    }
+
+#else
+    GTEST_SKIP();
+#endif
+};
+
+TEST_P(ConfigWithBFloat16, BFloat16Test)
+{
+#if MIOPEN_EMBED_DB
+
+    const auto& handle = get_handle();
+    if(miopen::StartsWith(handle.GetDeviceName(), "gfx908") || // Explicitly disabled
+       miopen::StartsWith(handle.GetDeviceName(), "gfx90a") || // Explicitly disabled
+       miopen::StartsWith(handle.GetDeviceName(), "gfx94") ||  // Implicitly disabled by default
+       miopen::StartsWith(handle.GetDeviceName(), "gfx103") || // Implicitly disabled by default
+       miopen::StartsWith(handle.GetDeviceName(), "gfx110") || // Implicitly disabled by default
+       GetFloatArg() != "--bfloat16")
+    {
+        GTEST_SKIP();
+    }
+    else
+    {
+        Run2dDriver(miopenBFloat16);
+    }
+
+#else
+    GTEST_SKIP();
+#endif
+};
+
+std::vector<std::string> GetTestCases(const std::string& precision)
+{
+#if MIOPEN_EMBED_DB
+    std::string flags = " --disable-validation --verbose ";
+#else
+    std::string flags = " ";
+#endif
```

Comment:
[R] conditional compilation is not necessary here.
```suggestion
    const std::string flags = " --disable-validation --verbose ";
```

Reply:
[Resolved]